apps/design-land/src/app/button/button.component.html

```diff
@@ -1,69 +1,36 @@
-<h1>Daff Button</h1>
+<daff-article>
+  <h1 daffArticleTitle>Button</h1>
+  <p daffArticleLead>
+    The button is used for making actions apparent to the end-user. It can be used to navigate users to a different page or to perform an action. Buttons can contain text, icons, or both.
+  </p>
 
-<h2>Basic Button</h2>
-<daff-button-set>
-  <button daff-button>Default</button>
-  <button daff-button color="primary">Primary</button>
-  <button daff-button color="secondary">Secondary</button>
-  <button daff-button color="tertiary">Tertiary</button>
-  <button daff-button color="black">Black</button>
-  <button daff-button color="white">White</button>
-  <button daff-button><fa-icon [icon]="faChevronLeft" daffPrefix></fa-icon>Button</button>
-  <button daff-button><fa-icon [icon]="faChevronRight" daffSuffix></fa-icon>Button</button>
-  <a href="#" daff-button>Link</a>
-  <button daff-button disabled>Disabled</button>
-</daff-button-set>
+  <h2>About</h2>
+  <p>
+    Native <code>&lt;button&gt;</code> or <code>&lt;a&gt;</code> elements are always used in order to provide an easy, accessible experience for users.
+  </p>
+  <ul>
+    <li><code>&lt;button&gt;</code> should be used when a clickable action is performed within the same page.</li>
+    <li><code>&lt;a&gt;</code> should be used for interactions that will navigate users to a new page or to a different target on the same page.</li>
+  </ul>
 
+  <h3>Basic Button</h3>
+  <design-land-example-viewer example="basic-button"></design-land-example-viewer>
+  
+  <h3>Stroked Button</h3>
```

Comment:
maybe this should be `[daff-stroked-button]`. Right now, I don't think there's any way to know how to use any of these components. How is a user supposed to know that "Stroked Button" translates to "daff-stroked-button"?

Reply:
The `stroked-button.component.html` tab has the usage example. It shows how to use a stroked button via <daff-stroked-button>.